packages/syntax-core/src/Box/Box.tsx

```diff
@@ -60,6 +60,13 @@ type Margin =
   | 12
   | "auto";
 type Padding = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+type Overflow =
```

Comment:
@christianvuerings it's possible that I went a little overboard on these. probably only _really_ need `visible` `hidden` `scroll` and `inherit`, but `auto` + `initial` seemed marginally useful as well?

do you have opinions on this?

Reply:
@somethiiing can we restrict it to:

```
hidden
scroll
auto
```